ui/src/PaperSummary.tsx

```diff
@@ -101,6 +101,7 @@ export class PaperSummary extends React.PureComponent<
           const hasMetrics = paper.citationVelocity !== 0 || paper.influentialCitationCount !== 0;
           const truncatedAbstract = paper.abstract ? truncateText(paper.abstract, 300) : null;
           const inLibrary = userLibrary ? userLibrary.paperIds.includes(this.props.paperId) : false;
+          const summaryUrl = `https://www.semanticscholar.org/paper/{paper.s2Id}`
```

Comment:
nit pick: maybe this name? 

```suggestion
          const paperUrl = `https://www.semanticscholar.org/paper/{paper.s2Id}`
```

Alternatively `pdpUrl` (short for 'paper details page URL') or `paperDetailsUrl` might make sense too! This is the naming convention used at Semantic Scholar to refer to the page at this URL.